recipes/pom.xml

```diff
@@ -117,6 +132,61 @@
             <artifactId>rewrite-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-api</artifactId>
+            <version>${camel.version}</version>
```

Comment:
Shouldn't all the Camel dependencies be of scope test?

Reply:
We are referencing them in the recipes (like using `*.class.getCanonicalName()` and similar). If we agree, that the camel dependencies have to stay in test scope only (which is probably the right decision), we will refactor recipes.

Reply:
fixed